### ISSUE TITLE
[wg/webfonts] WebFonts Working Group Recharter

### ISSUE THREAD

**Issue by svgeesus** [w3c/strategy#486]
New charter proposal, reviewers please take note.

# Charter Review

[Charter](https://w3c.github.io/charter-drafts/2025/webfonts-wg.html#background)

[diff to template](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fcharter-template.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2025%2Fwebfonts-wg.html%23background)
[diff to current charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2023%2F02%2Fwebfonts-2023.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2025%2Fwebfonts-wg.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/webfonts)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/orgs/w3c/projects/97/views/2).**

## Communities suggested for outreach

Font foundries and font vendors, particularly those specializing in fonts for languages with:

- large character repertoires (e.g. Chinese, Japanese)
- complex shaping (e.g. Indic, Arabic)

## Known or potential areas of concern

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)
This issue

## Anything else we should think about as we review? 

Note: proposed chairs should be copied @... on this issue.

@vlevantovsky @garretrieger

**Comment by himorin** [comment]
IFT has been merged into one specification (and received mass update following HR/WR) during current period, and I suppose we should mention about it in history table.

**Comment by svgeesus** [comment]
That wasn't a chartering change, but sure it can be mentioned in the history.

**Comment by ruoxiran** [comment]
no comment or request from APA.

**Comment by himorin** [comment]
No comment or request from i18n

**Comment by svgeesus** [comment]
Any comments from a privacy point of view? That is the one remaining HR needed before taking the proposed charter to TiLT.

**Comment by plehegar** [comment]
No comment from Privacy WG

**Comment by svgeesus** [comment]
Thanks @plehegar

**Comment by svgeesus** [comment]
AC Review started 17 Feb, ends 19 March.

**Comment by simoneonofri** [comment]
even if the review has already started, no comments from Security.

We're working on a generic File Format Threat Model, such as the one described [here](https://github.com/w3c/security-request/issues/59#issuecomment-2622054799):

> **PLS**: Parsing/Loading/Serializing
> **CD**: Compression/Decompression
> **EEC**: Embed Executable Code (e.g., scripts)
> **LER**: Links and external resources
> **MM**: Metadata manipulation
> **DI**: Data Integrity

**Comment by svgeesus** [comment]
> even if the review has already started, no comments from Security.

Thanks for confirming, @simoneonofri

**Comment by plehegar** [comment]
[Announced](https://lists.w3.org/Archives/Public/public-new-work/2025Mar/0005.html)
